src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar container__calendar calendar--month-length--29 calendar--start-day--tue">
```

Comment:
This classes is redundant
```suggestion
    <div class="calendar calendar--month-length--29 calendar--start-day--tue">
```